tests/foreman/cli/test_contentviews.py

```diff
@@ -692,39 +617,35 @@ def test_cv_associate_components_composite_negative(self):
             self.fail(err)
 
         # Associate repo to CV
-        result = ContentView.add_repository({u'id': new_cv['id'],
-                                             u'repository-id': new_repo['id']})
-        self.assertEqual(
-            result.return_code,
-            0,
-            "Repository was not associated to selected CV")
-        self.assertEqual(
-            len(result.stderr), 0, "No error was expected")
+        result = ContentView.add_repository({
+            u'id': new_cv['id'],
+            u'repository-id': new_repo['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0)
 
         # Publish a new version of CV
         result = ContentView.publish({u'id': new_cv['id']})
-        self.assertEqual(result.return_code, 0,
-                         "Publish CV was not successful")
+        self.assertEqual(
+            result.return_code, 0, "Publish CV was not successful")
```

Comment:
You have removed the third parameter, the error message that gets displayed to the user, in other parts of the code, so perhaps you should remove this one here too?